### PR TITLE
Add template cert

### PR DESCRIPTION
**Pull request by Anonymous4045**
Closes: #4

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Open+Sans|Pinyon+Script|Rochester');
```
Comment (interrrp) [inline template.html:7]: We could store a local copy of this font so it's faster

File: template.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```
Comment (interrrp) [inline template.html:5]: Noticed a lot of placeholder values, there should be something like `{name}` so it's easier to find and replace for the template generation

File: template.html [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        .pm-certificate-container {
+            position: relative;
+            width: 800px;
+            height: 600px;
+            background-color: #618597;
+            padding: 30px;
+            color: #333;
+            font-family: 'Open Sans', sans-serif;
+            box-shadow: 0 0 5px rgba(0, 0, 0, .5);
+            /*background: -webkit-repeating-linear-gradient(
```
Comment (interrrp) [inline template.html:67]: Remove dead code

File: template.html [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+                padding: 0;
+                border: 1px solid #E1E5F0;
+                background-color: rgba(255, 255, 255, 1);
+                background-image: none;
+                left: 50%;
+                margin-left: -360px;
+                top: 50%;
+                margin-top: -260px;
+
+                .pm-certificate-block {
```
Comment (interrrp) [inline template.html:117]: Love the Sass-style indentation, but this isn't Sass, unfortunately 😔

File: template.html [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+            <div class="pm-certificate-title cursive col-xs-12 text-center">
+                <h2>Let's Build A... Certificate of Shillfullness</h2>
+            </div>
+        </div>
+
+        <div class="row pm-certificate-body">
+            <div class="pm-certificate-block">
+                <div class="col-xs-12">
+                    <div class="row">
+                        <div class="col-xs-2"><!-- LEAVE EMPTY --></div>
```
Comment (interrrp) [inline template.html:197]: Who are you leaving these comments for?

**Review by interrrp — CHANGES_REQUESTED** [review]

File: template.html (hunk 5 above)
Comment (Anonymous4045) [reply template.html:197]: *I definitely didn't steal this template from somewhere else and strip off some of the unneeded code*

Comment (shenanigansd) [reply template.html:197]: _please cite your sources_

Comment (Anonymous4045) [reply template.html:197]: https://codepen.io/seebrosky/pen/xLzKOV

Comment (shenanigansd) [reply template.html:197]: Thank

**Comment by interrrp** [comment]
Oh it actually requires changes to merge, my bad

**Review by interrrp — APPROVED** [review]

**Pull request merged**